server/model/monitor.js

```diff
@@ -1526,6 +1543,19 @@ class Monitor extends BeanModel {
         ]);
     }
 
+    /**
+     * Gets Parent name
+     * @returns {Promise<string>} Name of this monitor's parent
+     */
+    async getParentName() {
+        if (this.parent === null) {
+            return "";
+        }
+
+        let parent = await Monitor.getParent(this.id);
+        return parent.name;
+    }
+
```

Comment:
Continiuation of the suggested changes in https://github.com/louislam/uptime-kuma/pull/4371/files#r1452821597

```suggestion
```